### PR TITLE
Update SetWeaponDamageModifier.md

### PR DESCRIPTION
**Pull request by Mojito-Fivem**
Added more detailed documentation for the native.

### CODE REVIEW

File: WEAPON/SetWeaponDamageModifier.md [hunk 1]
```diff
@@ -12,3 +12,19 @@ void _SET_WEAPON_DAMAGE_MODIFIER(Hash weaponHash, float damageAmount);
 ## Parameters
 * **weaponHash**: 
 * **damageAmount**: 
+
+```
```
Comment (AvarianKnight) [inline WEAPON/SetWeaponDamageModifier.md:16]: Shouldn't it be stated that it's required to run every frame?

Comment (AvarianKnight) [inline WEAPON/SetWeaponDamageModifier.md:16]: Shouldn't the description be above the parameters and the description for what the parameter does, under the parameters field?

Comment (Mojito-Fivem) [reply WEAPON/SetWeaponDamageModifier.md:19]: Ammended

File: WEAPON/SetWeaponDamageModifier.md [hunk 2]
```diff
@@ -8,13 +8,15 @@ aliases: ["0x4757F00BC6323CFE"]
 // 0x4757F00BC6323CFE
 void _SET_WEAPON_DAMAGE_MODIFIER(Hash weaponHash, float damageAmount);
 ```
+```
+Changes the weapon damage output by a given multiplier value. Must be ran every frame.
+```
 
 ## Parameters
```
Comment (AvarianKnight) [inline WEAPON/SetWeaponDamageModifier.md:15]: ## Parameters
* **weaponHash**: The hash of the weapon
* **damageAmount**:  Damage Multiplier

**Review by AvarianKnight — CHANGES_REQUESTED** [review]
I do believe they only use code fences on descriptions for NativeDB descriptions.

Also, shouldnt damageAmount be damageMultiplier?

**Review by AvarianKnight — APPROVED** [review]
Looks good

**Pull request merged**